### ISSUE TITLE
[DonationLogging:dpy2] 'DonoBank' object has no attribute 'lower'

### ISSUE THREAD

**Issue by Kuro-Rui** [i-am-zaidali/cray-cogs#27]
Here's the full traceback:
```py
Exception in command 'dono setup'
Traceback (most recent call last):
  File "C:\Users\User\T.D.L\lib\site-packages\discord\ext\commands\core.py", line 182, in wrapped
    ret = await coro(*args, **kwargs)
  File "C:\Users\User\AppData\Local\Red-DiscordBot\Red-DiscordBot\data\tdl\cogs\CogManager\cogs\donationlogging\dono.py", line 213, in setup
    await self.cache.set_default_category(ctx.guild.id, bank)
  File "C:\Users\User\AppData\Local\Red-DiscordBot\Red-DiscordBot\data\tdl\cogs\CogManager\cogs\donationlogging\models.py", line 326, in set_default_category
    if not (await self._verify_guild_category(guild_id, category))[0]:
  File "C:\Users\User\AppData\Local\Red-DiscordBot\Red-DiscordBot\data\tdl\cogs\CogManager\cogs\donationlogging\models.py", line 213, in _verify_guild_category
    org = category.lower() in categories.keys()
AttributeError: 'DonoBank' object has no attribute 'lower'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "C:\Users\User\T.D.L\lib\site-packages\discord\ext\commands\bot.py", line 1055, in invoke
    await ctx.command.invoke(ctx)
  File "C:\Users\User\T.D.L\lib\site-packages\redbot\core\commands\commands.py", line 759, in invoke
    await super().invoke(ctx)
  File "C:\Users\User\T.D.L\lib\site-packages\discord\ext\commands\core.py", line 1490, in invoke
    await ctx.invoked_subcommand.invoke(ctx)
  File "C:\Users\User\T.D.L\lib\site-packages\discord\ext\commands\core.py", line 899, in invoke
    await injected(*ctx.args, **ctx.kwargs)
  File "C:\Users\User\T.D.L\lib\site-packages\discord\ext\commands\core.py", line 191, in wrapped
    raise CommandInvokeError(exc) from exc
discord.ext.commands.errors.CommandInvokeError: Command raised an exception: AttributeError: 'DonoBank' object has no attribute 'lower'
```

**Comment by Kuro-Rui** [comment]
Looks like the setup is done, but the available subcommands for `dono` after setup are only:
`dono bal`, `dono leaderboard`, and `dono roles`. (There are no `dono check` or `dono add` or anything else)

**Comment by Kuro-Rui** [comment]
^ For this, nvm, I forgot to add the manager role to myself. I just wonder about the traceback, sorry.

**Comment by i-am-zaidali** [comment]
This error has been fixed with 6475bac747ce487c3933441e90b3bede25f09860
And with major changes happening to dpy at the moment, I won't be working on the dpy2 branch until dpy itself is stabilized. And i dont recommend using this branch until then
